### DISCUSSION TITLE
I can't make a series - What am I doing wrong?

### DISCUSSION THREAD

**Discussion by ooexiaoo** [nunocoracao/blowfish#378] [Q&A]
I wanted to make a series, so I followed all the directions provided here at blowfish documents [here](https://nunocoracao.github.io/blowfish/docs/series/)

But when I run the server I get this error message in the browser
`render of "page" failed: "/home/exia/blog/exiagarden/themes/blowfish/layouts/_default/single.html:35:11": execute of template failed at <partial "series.html" .>: error calling partial: "/home/exia/blog/exiagarden/themes/blowfish/layouts/partials/series.html:7:22": execute of template failed at <sort (index .Site.Taxonomies.series (index .Params.series 0 | urlize)) "Params.series_order">: error calling sort: sequence must be provided failed to render pages: sequence must be provided `

`<code class="filename" style="white-space: pre-wrap; overflow-wrap: break-word; font-family: SFMono-Regular, Menlo, Monaco, Consolas, &quot;Liberation Mono&quot;, &quot;Courier New&quot;, monospace; color: rgb(238, 247, 138); font-size: 0.9rem; line-height: 1.5; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(39, 42, 54); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">/home/exia/blog/exiagarden/themes/blowfish/layouts/_default/single.html:35:11:</code><span style="color: rgb(0, 0, 0); font-family: Muli, system-ui, -apple-system, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, &quot;Noto Sans&quot;, &quot;Liberation Sans&quot;, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Noto Color Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(39, 42, 54); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;"></span><div class="highlight" style="overflow-x: auto; color: rgb(0, 0, 0); font-family: Muli, system-ui, -apple-system, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, &quot;Noto Sans&quot;, &quot;Liberation Sans&quot;, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Noto Color Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(39, 42, 54); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div style="color: rgb(248, 248, 242); background-color: rgb(40, 42, 54); tab-size: 4; display: grid;">

33 34 35 36 37 | {{ end }}       <div class="min-w-0 min-h-0 max-w-prose">         {{ partial "series.html" . }}         {{ .Content \| emojify }}         </br></br>
-- | --

</div></div>`
`<code class="filename" style="white-space: pre-wrap; overflow-wrap: break-word; font-family: SFMono-Regular, Menlo, Monaco, Consolas, &quot;Liberation Mono&quot;, &quot;Courier New&quot;, monospace; color: rgb(238, 247, 138); font-size: 0.9rem; line-height: 1.5; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(39, 42, 54); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">/home/exia/blog/exiagarden/themes/blowfish/layouts/partials/series.html:7:22:</code><span style="color: rgb(0, 0, 0); font-family: Muli, system-ui, -apple-system, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, &quot;Noto Sans&quot;, &quot;Liberation Sans&quot;, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Noto Color Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(39, 42, 54); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;"></span><div class="highlight" style="overflow-x: auto; color: rgb(0, 0, 0); font-family: Muli, system-ui, -apple-system, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, &quot;Noto Sans&quot;, &quot;Liberation Sans&quot;, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Noto Color Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(39, 42, 54); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div style="color: rgb(248, 248, 242); background-color: rgb(40, 42, 54); tab-size: 4; display: grid;">

5 6 7 8 9 | {{ index .Params.series 0 }} - This article is part of a series.     </summary>     {{ range $post := sort (index .Site.Taxonomies.series (index .Params.series 0 \| urlize)) "Params.series_order" }}     {{ if eq $post.Permalink $.Page.Permalink }}     <div
-- | --

</div></div>`

![Screenshot_10](https://user-images.githubusercontent.com/114328638/210507372-b78129e5-3189-4c9b-9c19-02efc13fbbb5.png)

I think I'm doing something wrong, but I'm not good enough to understand what is it! Any help would be appriciated

**Comment by nunocoracao** [comment]
Did you define the series_order front matter parameter?

**Reply by ooexiaoo** [reply]
Yes, I have!

After following the series creation documentation I have made the following changes -
config.toml
`[taxonomies]
  tag = "tags"
  category = "categories"
  author = "authors"
  series = "series"`

and Marked the article in the front matter
`---
title: 📚 Chapter 1 & 2 - Basic HTML & CSS
date: 2022-11-02T09:26:30+05:30
Categories: Project Notes
tags: HTML & CSS
series: ["HTML Notes"]
series_order: 1
---`

I also have series behavior set to open
`seriesOpened = true`

**Reply by nunocoracao** [reply]
@ooexiaoo could you point me to your repo? I would need to check it out directly in the code. What you have there doesn't seem to be a problem. Any errors when running Hugo?

**Reply by nunocoracao** [reply]
@ooexiaoo alternatively can you give me the command you are running, and the full frontmatter for the md page?

**Reply by ooexiaoo** [reply]
I'm still a newbie so I don't have a repo, but I've been running `hugo server` and `hugo` commands when I get the error that I've screen of above.

This is the front matter I run which causes the issue
`---
title: 📚 Chapter 1 & 2 - Basic HTML & CSS
date: 2022-11-02T09:26:30+05:30
Categories: Project Notes
tags: HTML & CSS
series: ["HTML Notes"]
series_order: 1
---`

**Reply by nunocoracao** [reply]
Can't reproduce it like that. Let me know when you put it on a repo and I can take a look.

**Reply by ooexiaoo** [reply]
Thanks...
I'll do that 👍🏻
